src/main/java/com/fasterxml/jackson/databind/util/TokenBuffer.java

```diff
@@ -1703,8 +1703,7 @@ public int rawType(int index)
             if (index > 0) {
                 l >>= (index << 2);
             }
-            int ix = ((int) l) & 0xF;
-            return ix;
+            return ((int) l) & 0xF;
```

Comment:
I would argue that this change is not that good.
Since at least the local variable explains what it does.